prow/config/tide_test.go

```diff
@@ -43,3 +45,40 @@ func TestTideQuery(t *testing.T) {
 	checkTok("-label:\"foo\"")
 	checkTok("review:approved")
 }
+
+func TestMergeMethod(t *testing.T) {
+	ti := &Tide{
+		MergeType: map[string]string{
+			"foo/bar":           github.MergeRebase,
+			"kubernetes/charts": github.MergeSquash,
+		},
+	}
+
+	var testcases = []struct {
+		org      string
+		repo     string
+		expected string
+	}{
+		{
+			"baz",
```

Comment:
nit: names would help give meaning to these tests, e.g. here an error like:

```
default to merge if unset: expected merge method merge but got squash
```

instead of just the latter part

Reply:
Fixed